[2025-04-01T19:30:55.962+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T19:30:55.972+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 manual__2025-04-01T19:30:55.187814+00:00 [queued]>
[2025-04-01T19:30:55.977+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 manual__2025-04-01T19:30:55.187814+00:00 [queued]>
[2025-04-01T19:30:55.978+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-01T19:30:56.000+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): task1> on 2025-04-01 19:30:55.187814+00:00
[2025-04-01T19:30:56.007+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'manual__2025-04-01T19:30:55.187814+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpngfc1a95']
[2025-04-01T19:30:56.009+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6259) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T19:30:56.010+0000] {standard_task_runner.py:72} INFO - Started process 6260 to run task
[2025-04-01T19:30:56.010+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask task1
[2025-04-01T19:30:56.097+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.task1 manual__2025-04-01T19:30:55.187814+00:00 [running]> on host 7d980c7fabe2
[2025-04-01T19:30:56.171+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='grypton' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T19:30:55.187814+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T19:30:55.187814+00:00'
[2025-04-01T19:30:56.173+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T19:30:56.343+0000] {python.py:240} INFO - Done. Returned value was: Hello Grypton from PythonOperator
[2025-04-01T19:30:56.548+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T19:30:56.551+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task1, run_id=manual__2025-04-01T19:30:55.187814+00:00, execution_date=20250401T193055, start_date=20250401T193055, end_date=20250401T193056
[2025-04-01T19:30:56.608+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T19:30:56.618+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T19:30:56.620+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
